usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/ValidationUsecase.java

```diff
@@ -0,0 +1,23 @@
+/*
+ *
+ *  * Copyright (c) 2020. MobilityData IO.
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package org.mobilitydata.gtfsvalidator.usecase;
+
+public interface ValidationUsecase {
```

Comment:
As not all use cases will implement this interface, it shall bear a name that point to its specificity. I suggest `FileSpecificUsecase`